morpheus/utils/downloader.py

```diff
@@ -124,6 +122,9 @@ def get_dask_client(self):
         """
         import dask.distributed
 
+        # Up the heartbeat interval which can get violated with long download times
+        dask.config.set({"distributed.client.heartbeat": self._dask_heartbeat_interval})
+
```

Comment:
Question: Why was this moved here? Seems like it would be better to keep it with the dask cluster creation code

Reply:
moved it to `get_dask_client` because it's a config specific to the dask client. from dask docs:
```
This value is the time between heartbeats The client sends a periodic heartbeat message to the scheduler. If it misses enough of these then the scheduler assumes that it has gone.
```